app/assets/javascripts/app/core/models/comment.js

```diff
@@ -3,9 +3,44 @@
 
   angular
     .module('council.core')
-    .controller('Comment', Comment);
+    .factory('DiscussionComment', DiscussionComment);
```

Comment:
I used a file that already existed but was not being used, but the name `DiscussionComment` was already given.